### DISCUSSION TITLE
GLB Export not working

### DISCUSSION THREAD

**Discussion by akutterxressential** [atteneder/glTFast#334] [Q&A] [answered]
Hello guys,

I can't succeed in exporting a GLB or glTF from Unity. I created a HDRP demo scene in Unity 2021.2.9f1, imported glTFast. Everytime I try to export a GLB or glTF file the resulting file is not working. You find a "video proof" and the files under this link:

https://cloud.vrinsight.de/index.php/s/e443zGSrTrm9ZZf

Can someone help me understanding the issue? Thank you very much and my regards

Alex

**Comment by akutterxressential** [comment]
Ah by the way, this error also occurs sometimes when doing the same procedure like in the video:

https://cloud.vrinsight.de/index.php/s/wktqYfYzHHPydaE

`{
    "uri": "Base.gltf",
    "validatorVersion": "2.0.0-dev.3.4",
    "validatedAt": "2022-02-17T13:02:28.838Z",
    "issues": {
        "numErrors": 1,
        "numWarnings": 0,
        "numInfos": 0,
        "numHints": 0,
        "messages": [
            {
                "code": "INVALID_JSON",
                "message": "Invalid JSON data. Parser output: FormatException: SyntaxError: Unexpected number in JSON at position 136",
                "severity": 0,
                "pointer": ""
            }
        ],
        "truncated": false
    }`

**Comment by atteneder** [comment, marked as answer]
@akuttervrinsightde Thanks for the report.

Some of the float values (e.g. of the translation/rotation) are serialized using `float.ToString("R")`. This is not culture-invariant, so assuming your machin is configured to German the output is incorrect (comma instead of decimal point)

How it is in your files:

```json
"translation": [
    -2,
    35910491E-07,
    1,
    84,
    2,
    68133533E-07
]
```

What should be:

```json
"translation": [
    -2.35910491E-07,
    1.84,
    2.68133533E-07
]
```

I'll fix it for the next release.

**Reply by atteneder** [reply]
@akuttervrinsightde For obvious problems/bugs, please create an issue instead next time. Discussions are more for feature and workflow brain-storming.
